### DISCUSSION TITLE
New Source: Trino

### DISCUSSION THREAD

**Discussion by kmanamcheri** [airbytehq/airbyte#37387] [New Connector Request]
Trino (https://trino.io/) is a distributed query engine for big data analytics (similar to Presto, and Hive). I would like to add Trino as a source connector to Airbyte. Would this be something desirable? Is there a reason why there aren't any big data open-source query engines such as Trino, Presto and Hive not available in Airbyte?

**Comment by natikgadzhi** [comment]
Honestly not sure — but a source would be very welcome for sure.

**Comment by natikgadzhi** [comment]
One caveat is that you'd have to deal with dynamic schemas, so that would probably be a Python CDK source, not a low-code one.

[a comment by an automated account is not shown]
